code/espurna/ntp.h

```diff
@@ -0,0 +1,52 @@
+/*
+
+NTP MODULE
+
+*/
+
+#pragma once
+
+#include "broker.h"
+
+// TODO: need this prototype for .ino
+struct NtpCalendarWeekday;
+
+#if NTP_SUPPORT
```

Comment:
BTW some benefit of having `#if` guards in headers is that compilation will fail instead of linker when function is not implemented / supported